operator/redisfailover/service/client.go

```diff
@@ -85,8 +88,11 @@ func (r *RedisFailoverKubeClient) EnsureRedisStatefulset(rf *redisfailoverv1alph
 
 // EnsureRedisConfigMap makes sure the sentinel configmap exists
 func (r *RedisFailoverKubeClient) EnsureRedisConfigMap(rf *redisfailoverv1alpha2.RedisFailover, labels map[string]string, ownerRefs []metav1.OwnerReference) error {
-	cm := generateRedisConfigMap(rf, labels, ownerRefs)
-	return r.K8SService.CreateOrUpdateConfigMap(rf.Namespace, cm)
+	if _, err := r.K8SService.GetConfigMap(rf.Namespace, GetSentinelName(rf)); err != nil {
+		cm := generateRedisConfigMap(rf, labels, ownerRefs)
```

Comment:
This should check first if the custom ConfigMap is set (hence the user wants to use it).